### ISSUE TITLE
Contact form in footer

### ISSUE THREAD

**Issue by charlieforward9** [cameron-keene/ACE_Website#19]
Prevent whitespace problem below footer on Admin Dashboard
<img width="1354" alt="Screen Shot 2021-11-02 at 6 48 31 PM" src="https://user-images.githubusercontent.com/62311337/139962337-d07912af-e977-446a-98db-5d6fae8037eb.png">

**Comment by charlieforward9** [comment]
Removing menu from the footer, adding a contact form in the footer.

**Comment by charlieforward9** [comment]
This issue can be closed once the functionality is merged into main @cameron-keene

**Comment by charlieforward9** [comment]
This will look better if we condense the first and last names together. Two rows instead of three

![Screen Shot 2021-11-08 at 10 02 45 PM](https://user-images.githubusercontent.com/62311337/140854284-9a7a5182-b869-4c16-a188-65ffd8fcb6ec.png)

I took care of this.

**Comment by charlieforward9** [comment]
@cameron-keene @blakerand where does the form info go to when it is submitted?
